website/src/categories/getting-started/developers.md

```diff
@@ -20,6 +20,23 @@ eleventyNavigation:
 
 <section>
 
+## Make sure you have access
+
+Even though the SL Design System is open source, the packages are not. That is why you need to get access to the npm packages in order to install them locally. First the SL Design System team needs to add you to the list of users who can access the packages. To be added to that list you can <a href="mailto:designsystem@sanoma.com">send us an email</a> or reach out to us in [our Slack channel](https://sanoma.slack.com/archives/C03SA9HUUA3).
```

Comment:
"First the SL Design System team needs your to add you to a team within our GitHub organisation. To be added, you need to provide your `@sanoma.com` (or other company) email address to us in Slack (...) or mail us at ....."